File: Beecrowd/C/1007.c
#include <stdio.h>

int diferenca(int a, int b, int c, int d);

int main() {
    int A, B, C, D;
    scanf("%d%d%d%d", &A, &B, &C, &D);
    printf("DIFERENCA = %d\n", diferenca(A, B, C, D));

    return 0;
}

int diferenca(int a, int b, int c, int d) {
    return a*b-c*d;
}

File: Beecrowd/JavaScript/1007.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const a = parseInt(lines.shift());
const b = parseInt(lines.shift());
const c = parseInt(lines.shift());
const d = parseInt(lines.shift());

console.log(`DIFERENCA = ${a*b - c*d}`);
